packages/gestalt/src/Masonry/defaultLayout.ts

```diff
@@ -99,9 +99,10 @@ const defaultLayout =
           ...otherProps,
         })
       : items.map((item) => {
-          const height = isLoadingStateItem(item, renderLoadingState)
-            ? item.height
-            : measurementCache.get(item);
+          const height =
```

Comment:
thoughts on making sure that the pin is a skeleton pin? we could just check to see if `height` is available, but i wasnt sure if we wanted to expose that API